setup.py

```diff
@@ -97,6 +97,7 @@
     'pytz>=2015.4',
     'pyyaml==3.11',
     'requests==2.9.1',
+    'scikit-learn==0.17.1',
     'ipykernel==4.4.1',
     'psutil==4.3.0',
```

Comment:
Unrelated to your changes, `'psutil==4.3.0',` appears twice in this list. Could it be fixed in this PR or should we create a separate issue?